lib/matplotlib/rcsetup.py

```diff
@@ -1012,7 +1015,7 @@ def _validate_linestyle(ls):
     'lines.solid_joinstyle': ['round', validate_joinstyle],
     'lines.dash_capstyle':   ['butt', validate_capstyle],
     'lines.solid_capstyle':  ['projecting', validate_capstyle],
-    'lines.dashed_pattern':  [[3.7, 1.6], validate_nseq_float(allow_none=True)],
+    'lines.dashed_pattern': [[3.7, 1.6], validate_nseq_float(allow_none=True)],
```

Comment:
While generally, we try to be flake8 conform, I would not insist on line length limits for the definition of the default parameters. These definitions should be kept as structured as possible. I thus propose to leave E501 in and limit the changes to `defaultParams` to cases where we do not have to bent over or excessively stretch comments over serveral lines to satisfy the line limit.